pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -1318,7 +1320,7 @@ func (m *kubeGenericRuntimeManager) SyncPod(ctx context.Context, pod *v1.Pod, po
 	// Step 7: For containers in podContainerChanges.ContainersToUpdate[CPU,Memory] list, invoke UpdateContainerResources
 	if isInPlacePodVerticalScalingAllowed(pod) {
 		if len(podContainerChanges.ContainersToUpdate) > 0 || podContainerChanges.UpdatePodResources {
-			m.doPodResizeAction(pod, podStatus, podContainerChanges, result)
+			m.doPodResizeAction(pod, podStatus, podContainerChanges, &result)
```

Comment:
Why don't we let m.doPodResizeAction return an error and call result.Fail here uniformly? 
Something like this:
```suggestion
			if err := m.doPodResizeAction(pod, podStatus, podContainerChanges, &result); err != nil {
			        result.Fail(err)
			}
```

Reply:
It looks better. Thanks!